src/passwd.c

```diff
@@ -426,7 +426,7 @@ static void print_status (const struct passwd *pw)
 
 	sp = prefix_getspnam (pw->pw_name); /* local, no need for xprefix_getspnam */
 	if (NULL != sp) {
-		date_to_str (sizeof(date), date, sp->sp_lstchg * DAY),
+		date_to_str (sizeof(date), date, (time_t) sp->sp_lstchg * DAY),
```

Comment:
How about redefining DAY to use `long long` instead of adding casts?

<https://github.com/shadow-maint/shadow/pull/880>

Reply:
Removed the casts, so this PR basically depends on https://github.com/shadow-maint/shadow/pull/880

Reply:
:-)